Source/WebCore/platform/graphics/filters/FilterImage.cpp

```diff
@@ -87,6 +87,28 @@ FloatPoint FilterImage::mappedAbsolutePoint(const FloatPoint& point) const
     return FloatPoint(point - m_absoluteImageRect.location());
 }
 
+size_t FilterImage::memoryCost() const
+{
+    CheckedSize memoryCost;
+
+    if (m_imageBuffer)
+        memoryCost += m_imageBuffer->memoryCost();
+
+    if (m_unpremultipliedPixelBuffer)
+        memoryCost += m_unpremultipliedPixelBuffer->sizeInBytes();
+
+    if (m_premultipliedPixelBuffer)
+        memoryCost += m_premultipliedPixelBuffer->sizeInBytes();
+
+#if USE(CORE_IMAGE)
+    if (m_ciImage)
+        memoryCost += memoryCostOfCIImage();
+#endif
+
+    RELEASE_ASSERT(!memoryCost.hasOverflowed());
```

Comment:
Is this needed? I think `Checked`'s `size_t` conversion operator will crash for you. (Although I guess it won't have a useful assertion message?)